flutter/test/mocks.mocks.dart

```diff
@@ -1,5 +1,5 @@
-// Mocks generated by Mockito 5.0.0-nullsafety.7 from annotations
-// in sentry_flutter/test/mocks.dart.
+// Mocks generated by Mockito 5.0.3 from annotations
+// in sentry_flutter/example/ios/.symlinks/plugins/sentry_flutter/test/mocks.dart.
```

Comment:
This path seems funky. Why is there does the generator go through a symlink 😅

Reply:
Ha, good question 😅

Reply:
I don't think it's a problem. I just thought it was funny :D